lambda-template-java/src/test/java/com/tmobile/services/HandlerTest.java

```diff
@@ -14,23 +14,41 @@
 // limitations under the License.
```

Comment:
This is our template file that gets committed to the user repo when they create a Java service, so it wouldn't make sense to add unit tests that would fail once the user starts changing code.